Error packing product: Problem creating zip: Execution exception: Java heap space

This happens from time to time on the Jenkins CI servers.

```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-p2-director-plugin:2.7.0:archive-products (archive-products) on project org.eclipse.chemclipse.rcp.compilation.community.product.id: Error packing product: Problem creating zip: Execution exception: Java heap space -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :org.eclipse.chemclipse.rcp.compilation.community.product.id
```

--- a/.mvn/jvm.config
+++ b/.mvn/jvm.config
@@ -1,0 +1,1 @@
+-Xmx2048m -Xms1024m -XX:MaxPermSize=512m

